Run stomp planner error "The 'stomp' configuration parameter was not found"

I install moveit and stomp planner from source in Ubuntu 20. When i run the demo for the stomp planner with the command "roslaunch panda_moveit_config demo_stomp.launch ", i get the following error:
`[ERROR] [1674098279.924124491]: The 'stomp' configuration parameter was not found
terminate called after throwing an instance of 'std::runtime_error'
  what():  Unable to initialize planning plugin
[move_group-5] process has died [pid 3939941, exit code -6, cmd /home/dell/ws_moveit/devel/lib/moveit_ros_move_group/move_group --debug __name:=move_group __log:=/home/dell/.ros/log/df160252-97a7-11ed-b47c-ebb3863f028e/move_group-5.log].
log file: /home/dell/.ros/log/df160252-97a7-11ed-b47c-ebb3863f028e/move_group-5*.log
[ INFO] [1674098280.451824028]: Stereo is NOT SUPPORTED
[ INFO] [1674098280.451920599]: OpenGL device: NVIDIA GeForce RTX 3090/PCIe/SSE2
[ INFO] [1674098280.451941955]: OpenGl version: 4.6 (GLSL 4.6).
[ INFO] [1674098283.856437647]: Loading robot model 'panda'...
[ INFO] [1674098283.959639331]: Starting planning scene monitor
[ INFO] [1674098283.961705455]: Listening to '/move_group/monitored_planning_scene'
[ INFO] [1674098284.367928116]: waitForService: Service [/get_planning_scene] has not been advertised, waiting...
[ INFO] [1674098289.374384433]: Failed to call service get_planning_scene, have you launched move_group or called psm.providePlanningSceneService()?
[ INFO] [1674098289.374592777]: Constructing new MoveGroup connection for group 'panda_arm' in namespace ''
[ERROR] [1674098319.388580579]: Unable to connect to move_group action server 'move_group' within allotted time (30s)
[ INFO] [1674098319.389808524]: Constructing new MoveGroup connection for group 'panda_arm' in namespace ''
[ERROR] [1674098349.402577899]: Unable to connect to move_group action server 'move_group' within allotted time (30s)
`
All planner config are right, any solutions to deal with that?

--- a/launch/stomp_planning_pipeline.launch.xml
+++ b/launch/stomp_planning_pipeline.launch.xml
@@ -22,13 +22,15 @@
   <!-- <param name="capabilities" value="" /> -->
 
   <!-- Load parameters for both groups, panda_arm and manipulator -->
-  <group ns="$(arg arm_id)_arm">
-    <rosparam command="load" file="$(find panda_moveit_config)/config/stomp_planning.yaml" />
-    <param name="group_name" value="$(arg arm_id)_arm" />
-  </group>
-  <group ns="manipulator">
-    <rosparam command="load" file="$(find panda_moveit_config)/config/stomp_planning.yaml" />
-    <param name="group_name" value="manipulator" />
+  <group ns="stomp">
+    <group ns="$(arg arm_id)_arm">
+      <rosparam command="load" file="$(find panda_moveit_config)/config/stomp_planning.yaml"/>
+      <param name="group_name" value="$(arg arm_id)_arm"/>
+    </group>
+    <group ns="$(arg arm_id)_manipulator">
+      <rosparam command="load" file="$(find panda_moveit_config)/config/stomp_planning.yaml"/>
+      <param name="group_name" value="$(arg arm_id)_manipulator"/>
+    </group>
   </group>
 
 </launch>

